elastic/integrations
# update fields to include new cgroupsV2 fields

## What does this PR do?

This updates the integration with the fields changes that are already in Beats, namely cgroups V2 support.

## Checklist


## How to test this PR locally
- Pull down and build
- Run on a system running cgroups V2 (Like the latest fedora release)
- Make sure `system/process` collects all the data it should, and that all fields are documented.
